### PR TITLE
First take of implementation

### PR DESCRIPTION
**Pull request by obmarg**
See README for details basically.

### CODE REVIEW

**Comment by villaaston** [comment]
:kr:

File: README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+* `url` should contain a URL of the form "hostname:port" that logs should be
+  forwarded to.
+* `filter` should contain a valid syslog-ng filter expression.  Only logs that
+  match this filter will be forwarded to `url`.
+* `template` should contain the syslog-ng template to use for formatting log
+  messages.  This should include any tokens required by your log service.
+
+For example, to setup a log with etcdctl:
+
+    etcdctl set /deis/log/syslog-ng/my-app '{"url": "data.logentries.com:514", "filter": "program(my-app)", "template": "123456 $MSG"}''
```
Comment (villaaston) [inline README.md:31]: Is it worth saying that a newline is added to the end of the template here? Unless it doesn't matter if someone adds one in the template (and ends up with `...\n\n`)

Comment (obmarg) [inline README.md:31]: Guess it doesn't hurt to mention it.  The auto new-line was _pretty essential_ - logentries doesn't receive anything if you don't explicitly feed it a newline.

Comment (obmarg) [inline README.md:31]: (It took me 60+ minutes to figure that issue out)

**Comment by carlos-alberto** [comment]
```
if thisComment != nil {
    :shipit:
}
```

**Pull request merged**